libs/distancing.py

```diff
@@ -296,6 +298,10 @@ def process_video(self, video_uri):
                 frame_num += 1
                 if frame_num % 100 == 1:
                     logger.info(f'processed frame {frame_num} for {video_uri}')
+                if self.repeat == 'true' and frame_num % total_frames == 0:
```

Comment:
I think that the repeat parameter should be a boolean and init it using the function `get_boolean`  from the `config_engine`.

Reply:
Thanks. Done.